internal/server/data/data.go

```diff
@@ -111,6 +121,7 @@ func get[T models.Modelable](db *gorm.DB, selectors ...SelectorFunc) (*T, error)
 }
 
 func list[T models.Modelable](db *gorm.DB, selectors ...SelectorFunc) ([]T, error) {
+	db = db.Order(getDefaultSortFromType((*T)(nil)))
```

Comment:
Why not set default sort to ID instead?

Reply:
some people have asked for name to be the default, which is fine as long as you account for tables that don't have a name field.